testcases/kernel/syscalls/fdatasync/fdatasync03.c

```diff
@@ -19,24 +19,21 @@
 #include <sys/types.h>
 #include "tst_test.h"
 
-#define MNTPOINT	"mnt_point"
-#define FNAME		MNTPOINT"/test"
+#define FNAME		"/data/test"
 #define FILE_SIZE_MB	32
 #define FILE_SIZE	(FILE_SIZE_MB * TST_MB)
 #define MODE		0644
-#define dev            "/dev/vda"
+#define dev		"/dev/vdb"
```

Comment:
Not required now. Please add below config parameters to tst_test test structure initialization in fdatasync03.c  file.
.mount_device = 1,
.all_filesystems = 1,
.mntpoint = MNTPOINT,

By enabling above options, test framework will mount the file system at the time of test initialization time.

Reply:
@shaikshavali1 can you give more explanation. We are creating test image and mapping to /dev/vdb and /data in config. Why do we need to mount again? Why don't I need to specify /dev/vdb? Why do we need to mount again?